modules/gcp-vpn.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+//Specify the module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/gcp-private-connections.adoc
+
+[id="gcp-vpn_{context}"]
+= Configuring a GCP Cloud VPN
+
+[role="_abstract"]
+You can configure a {GCP} {product-title} cluster to securely connect your High Availability (HA) Virtual Private Network (VPN) to your Virtual Private Cloud (VPC) network through an IPsec tunnel.
+
+.Prerequisites
+* For peering a HA VPN to a peer gateway:
+** Your peer VPN gateway supports Border Gateway Protocol (BGP).
+** A GCP project with billing enabled.
+** An installed GCP SDK.
+** A VPC and at least one subnet in the region where the VPN gateway resides.
```

Comment:
>A VPC and at least one subnet in the region where the VPN gateway resides.

To make it more clear, use HA VPN gateway:
A VPC and at least one subnet in the region where the HA VPN gateway resides.